source/aggregation-pipeline-builder.txt

```diff
@@ -31,6 +31,13 @@ aggregation pipeline. The :guilabel:`Preview of Documents in the
 Collection` section shows 20 documents randomly sampled from the chosen
 collection.
 
+.. note:: 
+
+   When you connect Compass to MongoDB instances hosted on 
```

Comment:
[nit]

I'm admittedly forgetting the context around this decision, but I think we want to use "deployment" rather than "instance".

 ```suggestion
   When you connect Compass to a MongoDB deployment hosted on 
```